README.md

```diff
@@ -32,35 +32,35 @@ See [MainActivity in the sample app](https://github.com/jump-sdk/jetpack_compose
 Also check out the [ComposeCountryCodePicker documentation](https://jump-sdk.github.io/jetpack_compose_country_code_picker_emoji/ccp/com.togitech.ccp.component/-togi-country-code-picker.html) for all available composables and utilities.
 
 ```kotlin 
-        var phoneNumber = rememberSaveable { mutableStateOf("") }
-        var fullPhoneNumber = rememberSaveable { mutableStateOf("") }
-        var isNumberValid: Boolean by rememberSaveable { mutableStateOf(false) }
-
-        TogiCountryCodePicker(
-            onValueChange = { (code, phone), isValid ->
-                Log.d("CCP", "onValueChange: $code $phone -> $isValid")
-                
-                phoneNumber.value = phone
-                fullPhoneNumber.value = code + phone
-                isNumberValid = isValid
-            },
-        )
+var phoneNumber = rememberSaveable { mutableStateOf("") }
+var fullPhoneNumber = rememberSaveable { mutableStateOf("") }
+var isNumberValid: Boolean by rememberSaveable { mutableStateOf(false) }
+
+TogiCountryCodePicker(
+    onValueChange = { (code, phone), isValid ->
+        Log.d("CCP", "onValueChange: $code $phone -> $isValid")
+        
+        phoneNumber.value = phone
+        fullPhoneNumber.value = code + phone
+        isNumberValid = isValid
+    },
```

Comment:
```
val phoneNumber = rememberSaveable { mutableStateOf("") }
val fullPhoneNumber = rememberSaveable { mutableStateOf("") }
val isNumberValid = rememberSaveable { mutableStateOf(false) }

TogiCountryCodePicker(
    onValueChange = { (code, phone), isValid ->
        Log.d("CCP", "onValueChange: $code $phone -> $isValid")

        phoneNumber.value = phone
        fullPhoneNumber.value = code + phone
        isNumberValid.value = isValid
    },
)
```

`val` instead of `var`